truffle-config.js

```diff
@@ -85,7 +85,7 @@ module.exports = {
                     return new HDWalletProvider(mnemonic, 'http://34.67.242.174:8545')
             },
             network_id: 5,
-            gas: 8000000,
+            gas: 4000000,
```

Comment:
You are reverting your own change.

Make sure you merge the changes from the latest master in your branch to avoid such issues.

Reply:
Sorry about that.